tgui/packages/tgui/styles/main.scss

```
@@ -72,6 +72,7 @@
 @include meta.load-css('./interfaces/GamePreferences.scss');
 @include meta.load-css('./interfaces/DirectorPanel.scss');
 @include meta.load-css('./interfaces/PersonalCrafting');
+@include meta.load-css('./interfaces/AdminLogViewer.scss');
```

Stylelint (17.14.0)

[error] Unexpected extension ".scss" in `@meta.load-css` (scss/load-partial-extension)

(scss/load-partial-extension)